src/blog/_posts/2023-06-08-survey-results.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We released a user survey earlier this year to learn about who our users are, how they use Cadence, and how we can help them. After collecting the feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable and it helps us shape our roadmap for the future.
+
```

Comment:
I would add a line about your survey methodology, I assume this is mostly gathered from slack however. I get that we're not in the business of doing scientific-surveys, but it helps contextualize the discussion and limitations from what we can understand here.

Reply:
Added